src/main/scala/org/broadinstitute/dsde/rawls/model/VaultModel.scala

```diff
@@ -0,0 +1,17 @@
+package org.broadinstitute.dsde.rawls.model
+
+import com.wordnik.swagger.annotations.{ApiModelProperty, ApiModel}
+
+import scala.annotation.meta.field
+
+@ApiModel(value = "VaultEntity")
+case class VaultEntity(
+  @(ApiModelProperty@field)(required = true, value = "The id of this Vault object")
+  id: String,
+  @(ApiModelProperty@field)(required = true, value = "The URI to this Vault object")
+  getURL: String,
```

Comment:
i know its nitpicking, just might as well be consistent.  on that same note, I like to make all my object variables Object because everything extends that, that way I'm never wrong about the type :+1: